### ISSUE TITLE
Remove ticket items from alpine-fetch

### ISSUE THREAD

**Issue by githubbob42** [githubbob42/mingle2github2#3942]
**[Mingle Card: 4269](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/4269)**
Narrative
---------

Once ticket items are moved to the incremental sync, we should remove ticket items from the alpine-fetch sync (to avoid double syncing).

Acceptance Criteria
-------------------

Once this story is complete, ticket items and their associated metadata should be ONLY downloaded via the incremental sync.

_Analysis_
----------

_Note: all of the other stories in this feature are already handling validation and layouts so they are no longer needed in the fetch project._

*   _This story should be accepted at the same time as #4220._
*   _The metadata should be left in place temporarily because they are needed for mapping old events sitting in the queue during upversion._
*   _The ticket item fieldsets should be left in place until there is an acceptable replacement in the new sync._
*   _The subquery for ticket items needs to be stripped from the ticket step._
*   _The layouts and validation rules need to be stripped from ticket item metadata step._

Related Stories
---------------

Tasks
-----

{{table query: SELECT Number, Name, Owner, 'Task Status' WHERE Type = Task and Story = THIS CARD}}

Defects
-------

{{ table query: SELECT Number, Name, Owner, 'Status' WHERE Type = Defect and 'Related Story' = THIS CARD }}

Test Plan
---------

**This requires automated (Selenium) testing.  **

**Comment by githubbob42** [comment]
### Feature: #3862 Incremental Sync for Ticket Item ([Mingle](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/4189))

**Comment by githubbob42** [comment]
**Pull Request #97**
